File: practise/two.cpp
//simple class example
#include <iostream>
using namespace std;

class Student {
private:
	int rollNo;
	string name;

public:
	void setData(int r, string n){
		rollNo = r;
		name = n;
	}

	void display(){
		cout << "Roll no: " << rollNo << endl;
		cout << "Name: " << name << endl;
	}
};

int main(){
	Student s1;
	s1.setData(101, "Avinash");
	s1.display();
	return 0;
}

File: practise/two.rb
class Student
	def initialize
		@roll_no = nil
		@name = nil
	end

	def set_data(r, n)
		@roll_no = r
		@name = n
	end

	def display
		puts "Roll no: #{@roll_no}"
		puts "Name: #{@name}"
	end
end

s1 = Student.new
s1.set_data(101, "Avinash")
s1.display
